src/Symfony/Component/Validator/Tests/Constraints/FileValidatorPsr7Test.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Constraints;
+
+use Symfony\Component\Validator\Constraints\File;
+use Symfony\Component\Validator\Constraints\FileValidator;
+
+class FileValidatorPsr7Test extends AbstractConstraintValidatorTest
+{
+    protected function createValidator()
+    {
+        return new FileValidator();
+    }
+
+    public function testValidUploadedFile()
```

Comment:
There is not one FileValidatorTest, there are 2: `FileValidatorObjectTest`, `FileValidatorPathTest`, both extending `FileValidatorTest`. It makes no sense to put it in `FileValidatorTest`, as it would be run twice with exact the same input/output.